src/GeneralStateTestsFiller/stEWASMTests/log1TopicFiller.yml

```diff
@@ -0,0 +1,93 @@
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: ''
+      nonce: ''
+      storage: {}
+    # called contract, very simple, just calls log
+    deadbeef00000000000000000000000000000000:
+      balance: '100000000000'
+      code: |
+        (module
+          (import "ethereum" "log" (func $log (param i32 i32 i32 i32 i32 i32 i32)))
+          (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+          (memory 1)
+          (data (i32.const 0) "\de\ad\be\e0")
+          (data (i32.const 4) "\de\ad\be\e1")
+          (data (i32.const 8) "\de\ad\be\e2")
+          (data (i32.const 12) "\de\ad\be\e3")
+          (data (i32.const 16) "\de\ad\be\e4")
+          (data (i32.const 20) "\de\ad\be\e5")
+          (data (i32.const 24) "\de\ad\be\e6")
+          (data (i32.const 128) "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00")
+          (data (i32.const 160) "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\ff")
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+          (func $main
+            (local $storageKeyOffset i32)
+            (local $storageValueOffset i32)
+
+            (set_local $storageKeyOffset (i32.const 128))
+            (set_local $storageValueOffset (i32.const 160))
+
+            (call $log
+              ;; dataOffset i32ptr the memory offset to load data from (bytes)
+              (i32.const 0)
+              ;; length i32 the data length
+              (i32.const 0x000000ff)
+              ;; numberOfTopics i32 the number of topics following (0 to 4)
+              (i32.const 4)
+              ;; topic1 i32ptr the memory offset to load topic1 from (u256)
+              (i32.const 8)
+              ;; topic2 i32ptr the memory offset to load topic2 from (u256)
+              (i32.const 0xffffffff)
+              ;; topic3 i32ptr the memory offset to load topic3 from (u256)
+              (i32.const 0xffffffff)
+              ;; topic4 i32ptr the memory offset to load topic4 from (u256)
+              (i32.const 0xffffffff)
+            )
+
+            ;; Store a value to storage to ensure that the log completed without trapping
+            (call $storageStore (get_local $storageKeyOffset) (get_local $storageValueOffset))
+          )
+        )
+      nonce: ''
+      storage: {}
+  expect:
+    - indexes:
+        data: !!int -1
+        gas: !!int -1
+        value: !!int -1
+      network:
+        - ALL
+      result:
+        a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          # start_gas    - tx    - call - log - logData  * length - getReturnDataSize - storageStoreChange * 2
+          # 100000000000 - 21000 - 700  - 375 - 8        * 255    - 2                 - 5000 * 2
+          balance: '99999965883'
+        deadbeef00000000000000000000000000000000:
+          storage: {
+            # length of return data (zero)
```

Comment:
seems the comment is still incorrect, now that you have just one contract it is storing the hardcoded value `0x000...ff` instead of the return data.

Reply:
Yeah, I changed the original test case without removing/changing some of the comments.  Will update it and push filled tests after I finish fixing the gas calcs.